src/data/person.ts

```diff
@@ -67,3 +70,29 @@ export const deletePerson = (hass: HomeAssistant, personId: string) =>
     type: "person/delete",
     person_id: personId,
   });
+
+const cachedUserPerson: Record<string, string> = {};
```

Comment:
I put this mini-cache in given that we would hopefully not want to iterate the entire `states` array re-searching for it on every `hass` update, though now I realize we will still do that each update if there is no person for the logged in user.

Reply:
still better than no cache